apps/web/src/pages/index.tsx

```diff
@@ -9,30 +10,46 @@ import Twitter from 'src/components/Home/Twitter'
 import VisitAlternate from 'src/components/Home/VisitAlternate'
 import { Meta } from 'src/components/Meta'
 import { AuctionFragment } from 'src/data/subgraph/sdk.generated'
-import { getHomeLayout } from 'src/layouts/HomeLayout'
+import { DefaultLayout } from 'src/layouts/DefaultLayout'
+import { HomeLayout } from 'src/layouts/HomeLayout'
+import { LayoutWrapper } from 'src/layouts/LayoutWrapper'
 import { DaoFeed } from 'src/modules/dao'
+import Dashboard from 'src/modules/dashboard/Dashboard'
 
 import { NextPageWithLayout } from './_app'
 
 export type DaoProps = AuctionFragment['dao']
 
 const HomePage: NextPageWithLayout = () => {
+  const { address } = useAccount()
+
+  if (address) {
```

Comment:
imo if we have the `dashboard` button I think its okay to remove from the home page. like you mentioned it kinda feels like unexpected behavior and could confuse newer users. power users can easily bookmark / quickly nav to the dashboard

Reply:
For this one, I'd really like to defer to @isaaczaak , since he had requested auto-login in the product brief. 

https://hackmd.io/RzVQUHNQRFmhavYJ_3DVOw

I can see pros/cons to each approach. All are pretty easy to change or implement.

Reply:
discussed with @jordanlesich on this. i think it's okay for us to ship this and iterate on the experience.